web/src/modules/topic/components/Node/EditableNode.styles.tsx

```diff
@@ -64,7 +68,7 @@ const options = {
 
 /* some copied from https://github.com/wbkd/react-flow/blob/147656b22f577bb4141664d000e62ada9b490473/src/theme-default.css#L42-L77 */
 export const NodeDiv = styled("div", options)<NodeDivProps>`
-  width: ${nodeWidth}px;
+  width: clamp(200px, ${nodeWidthRem}rem, ${nodeWidthPx}px);
```

Comment:
nodeWidthRem = nodeWidthPx at 16 px default font size, so this clamp is actually making the width not scale for bigger default font sizes. I believe this is the reason for the very awkward problem node in the flow diagram
![image](https://user-images.githubusercontent.com/13872370/232667822-9d8daae3-235f-475c-9ae8-fec77cdb39c8.png)

additionally, the 200px minimum is smaller than nodeWidthRem * 16 = 176px, so we're clamping even at the default font size, making the flow nodes wider than they should be

Just to be sure - are you intentionally clamping this? The buttons & font size within the node are not clamped in either direction, so the size of the node itself does not remain consistent with these outside of the bounds

Reply:
`nodeWidthRem = nodeWidthPx at 16 px default font size, so this clamp is actually making the width not scale for bigger default font sizes. I believe this is the reason for the very awkward problem node in the flow diagram`
That is correct. I totally  missed the problem node having 3 buttons.

I can add a clamp to the buttons and text to fit the node at maxWidth, and not allow additional scaling.

the 200px was intentional, I can set it to the browser Default min